### DISCUSSION TITLE
Game crash when turning off fast forward in gbc titles

### DISCUSSION THREAD

**Discussion by davidlukerice** [eduardofilo/RG350_adam_image#113] [RetroArch topics]
Not sure if this is a retroarch issue or adam issue. I'm using an RG 280v with Adam 1.3 and when playing a gameboy color title, like pokemon crystal, if I use the R1 button to fast forward, about 80% of the time when I let the R1 button go to turn off fast forward, the game will crash and I have to hold down the power button to turn off the device.

**Comment by eduardofilo** [comment]
It looks like a RetroArch problem. I have tried updating to yesterday's distribution (2021-11-27) and completely resetting the configuration in case it had to do with some of the adjustments I made in Adam, and it keeps happening. Sorry, I can't do anything, it's a core Gambatte problem for sure. The only thing I can tell you is that you don't need to restart the console when the problem happens. Pressing Power + Select should close the hanging RetroArch session.
